bin/triton-translate.cpp

```diff
@@ -3,17 +3,20 @@
 #include "mlir/IR/AsmState.h"
 #include "mlir/IR/BuiltinOps.h"
 #include "mlir/IR/Dialect.h"
-#include "mlir/Parser.h"
+#include "mlir/Parser/Parser.h"
```

Comment:
need to update here https://github.com/openai/triton/blob/cf0ae2ed7623ccfac0447d0e61280e5f2a0cf77f/python/src/triton.cc#L11 as well?

Reply:
Fixed.